addons/better-img-uploads/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Better image uploads",
+  "description": "Automatically converts uploaded images into svg images to keep them from losing quality.",
+  "tags": ["editor", "beta"],
+  "credits": [
+    {
+      "name": "ErrorGamer2000",
+      "link": "https://scratch.mit.edu/users/ErrorGamer2000/"
+    }
+  ],
+  "enabledByDefault": true,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"],
+      "runAtComplete": true
+    }
+  ],
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "l10n": false
```

Comment:
True

Reply:
??? Make the `l10n` true?

Reply:
Yes, every addon should have this true

Reply:
even tho you don't use `msg`, l10n needs to be true so the addon name and description can be translated.